asm/dolphin/mtx/mtxvec.s

```diff
@@ -2,8 +2,8 @@
 
 .section .text  # 0x80342AA8 - 0x80342B4C
 
-.global PSMTXMUltiVec
-PSMTXMUltiVec:
+.global PSMTXMultiVec
```

Comment:
PSMTXMUltiVec is the actual name they used, lol

Reply:
Never mind, I don't see either spelling in a debug map, although I see it in other games' decomp repos. Probably just a misspelled name we came up with that got copy/pasted elsewhere?